dataactvalidator/config/sqlrules/d20_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,12 @@
+-- FundingAgencyCode is an optional field, but when provided must be a valid 3-digit CGAC agency code.
+SELECT
+    row_number,
+    funding_agency_code
+FROM detached_award_financial_assistance AS dafa
```

Comment:
This is still not aliasing everything. In the select you should have things like `dafa.row_number` and in the WHERE all the variables should be related to an alias (like `dafa.submission_id`)